### DISCUSSION TITLE
Dịch Interpretability, explainability.

### DISCUSSION THREAD

**Discussion by lkhphuc** [mlbvn/handson-ml2-vn#42] [Terminology] [answered]
Hai từ này khá giống nhau, hay được đi chung hoặc nhiều khi được sử dụng thay thế, nhưng tuỳ trường hợp lại có thể hàm ý hơi khác nhau.
Mình đọc lướt qua cũng không rõ trong sách này đang xài 2 từ này cùng một nghĩa hay khác nghĩa, nhưng để thống nhất mình đề xuất:
- `interpretation / interpretability`: `diễn dịch / khả năng diễn dịch`
- `explaination /  explainability`: `giải thích / khả năng giải thích`.

Mention @mlbvn/handson-ml2

**Comment by minhduc0711** [comment]
có vẻ 2 từ này trong sách được mang ý nghĩa tương đương thì phải: "hiểu tại sao mô hình lại đưa ra dự đoán đó"

**Comment by minhduc0711** [comment, marked as answer]
chốt nhé mọi người
- `interpretability`: `khả năng diễn dịch`
- `explainability`: `khả năng giải thích`
